meta-refkit-industrial/recipes-ros/cv-bridge/cv-bridge_%.bbappend

```diff
@@ -0,0 +1,3 @@
+# TODO: This way cv_bridge doesn't require libboost_python, but this tweak
+#       should be dropped after ROS is compatible with python3-only distro.
+EXTRA_OECMAKE_append_refkit-industrial = " -DANDROID=ON"
```

Comment:
Does this mean #109 isn't absolutely necessary for this anymore? What are the limitations after this change?